### ISSUE TITLE
release schedule

### ISSUE THREAD

**Issue by minemR** [Rdatatable/data.table#5385]
Whats the planned release schedule? Where one can find information related to this?
Whats the major bug found in:?

> That's a mistake in the vignette. If you check the news you'll find the entry under the current dev version, so it'll be included in the next release (barring the discovery of a major major bug). 1.14.2 was basically just a hotfix to accommodate a change in R's tooling. So that's likely how it happened.

_Originally posted by @tlapak in https://github.com/Rdatatable/data.table/issues/5369#issuecomment-1097188942_

Is this kind of information available openly?

**Comment by avimallu** [comment]
All `data.table` development is public AFAIK. There was no major bug - the comment you've mentioned states that there will be another release *if* there is a major new bug found prior to the release of the feature it is referring to.

**Comment by jangorecki** [comment]
IMO it make sense to release 1.14.4 as of now (including pending dev regressions). There are many big PRs waiting to be merge and it is bad idea to merge them shortly before releasing to CRAN.

**Comment by minemR** [comment]
Sorry, let me rephrase: Why there hasn't been a release from (27 Sep 2021) until now?
There are 40 NEW FEATURES, 53 BUG FIXES and 16 NOTES.
Is there a particular reason for that?

**Comment by jangorecki** [comment]
I think the only reason is the amount of work it is required for each cran release. Mainly revdeps testing, and follow up of it (fix regression, improve backward compatible, fix revdep code, etc.).

**Comment by grantmcdermott** [comment]
As a long-term user and beneficiary of data.table, I can't help but feel we're well into uncomfortable territory atm. Official release aside, I genuinely can't recall such a long pause in activity on the dev version (4+ months as I write this). 

I'm *deeply* appreciative of maintainer time and this incredible package that we've been able to reply on for so long. But I doubt I'm alone in feeling concerned about the unusual radio silence and pause in development. I would love some reassurance on that front!

**Comment by stefanfritsch** [comment]
Let's not turn this in a doom and gloom session. I, for my part, and I think minemR as well, are happy with the features in the dev version - we'd just like a release version with those features.

**Comment by jangorecki** [comment]
Release schedule has been discussed internally, and conclusions are:
- ASAP release of hotfix patch release 1.14.10 to CRAN
- shortly after release 1.15.0
- continue major releases 1.16.0, 1.17.0, ... in around 6 month cycles.

**Comment by mmaechler** [comment]
`ASAP` .. should really happen ASAP -- please !!
